crates/forge_analyzer/src/definitions.rs

```diff
@@ -1000,15 +1017,35 @@ impl FunctionAnalyzer<'_> {
                     && Some(&ImportKind::Default)
                         == self.res.is_imported_from(def, "@forge/api") =>
             {
+                let first_arg = first_arg?;
+                let is_as_app = authn.first() == Some(&PropPath::MemberCall("asApp".into()));
+
                 let function_name = if *last == "requestJira" {
-                    IntrinsicName::RequestJira
+                    // Resolve Jira API requests to either JSM/JS/Jira as all are bundled within requestJira()
+                    match first_arg {
+                        Expr::Tpl(template) => {
+                            let url = template
+                                .quasis
+                                .iter()
+                                .map(|quasi| quasi.raw.as_str())
+                                .collect::<String>();
+
+                            resolve_jira_api_type(&url).unwrap_or_else(|| {
+                                warn!("Falling back to generic Jira request");
+                                IntrinsicName::RequestJira // TODO: how should we handle this edge case?
+                            })
+                        }
+                        _ => {
+                            warn!("First parameter to requestJira() is invalid");
+                            IntrinsicName::RequestJira // TODO: how should we handle this edge case?
```

Comment:
If we want to produce the least amount of false positives, then we should conservatively assume it can use any of Jira APIs if we can't statically determine which one.

Reply:
Yeah that works.